### ISSUE TITLE
Rewrite CONTRIBUTING.md to be specific to this repo

### ISSUE THREAD

**Issue by Asartea** [TheOdinProject/theodinproject#3920]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `brief description of request` format, e.g. `Add dark mode to website`

The following checkbox is OPTIONAL:

-   [ ] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**
In #3901 we converted all links to `CONTRIBUTING.md` in the TOP org to point to the `.github` version, which is now the source of truth. That leaves the file in this repo now unnecessary in its current state, but it could be useful as a guide in how to contribute to this specific repo, and should be rewritten to fit that purpose

**2. Acceptance Criteria:**
- [ ] Decide upon what should be included, looking also at how the wiki currently fits together
- [ ] CONTRIBUTING.md has been rewritten with repo specific information

**Comment by CouchofTomato** [comment]
Yeah I agree. I don't have time for this at the moment though. See if anyone else wants to take it on.

**Comment by CBID2** [comment]
I’ll do it @CouchofTomato

**Comment by CarrieKroutil** [comment]
The `CONTRIBUTING.md` content has been updated per AC - suggest closing issue.
